symplyphysics/laws/waves/vector/phase_velocity_from_angular_velocity_and_wavevector.py

```diff
@@ -0,0 +1,74 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    Symbol,
+    Quantity,
+    Vector,
+    QuantityVector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    vector_magnitude,
+)
+
+# Description
+## The phase velocity of a wave is the rate at which the wave propagates in a medium.
+## It is the velocity at which the phase of one frequency component of the wave travels.
+## The phase velocity is collinear with the wavevector.
+
+# Law: v = w * k / |k|**2
+## v - vector of phase velocity of wave
+## w - angular frequency of wave
+## k - wavevector of wave
+## |k| - wavenumber of wave, where |v| is the Euclidean norm of vector v
+
+angular_frequency = Symbol("angular_frequency", angle_type / units.time, positive=True)
+
+
+def phase_velocity_law(wavevector_: Vector) -> Vector:
+    wavenumber_ = vector_magnitude(wavevector_)
+    return scale_vector(
+        angular_frequency / wavenumber_**2,
+        wavevector_,
+    )
+
+
+# k = |k| * e_k = (w / |v|) * (v / |v|) = w * v / |v|**2
```

Comment:
It does not look very obvious. Might be a good idea to add a proof, eg by taking some generic `w` and `v`, putting them into `phase_velocity_law` and checking the result.

Reply:
I'll add it below this law then